client/app/main/student/student.controller.js

```diff
@@ -14,6 +14,16 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr, Modal) {
     $scope.student = result.student;
     $scope.interventions = result.interventions;
     $scope.outreaches = result.outreaches;
+    Student.list({id: $stateParams.id}, function(data) {
+      var record = data.currentRecord.entries;
+      var enrolled = 0;
+      var present = 0;
+      _.forEach(record, function(entry) {
```

Comment:
Actually you probably want to use this: https://docs.angularjs.org/api/ng/filter/number

Then you would store the absence ratio on the controller:

```
$scope.attendancePercentage = entry.present/ entry.enrolled * 100;
```
and in the template:
```
{{ attendancePercentage | number: 1 }}
```

I think something like that.